### PR TITLE
branding & support related adjustments

### PR DESCRIPTION
**Pull request by kingthorin**
- Update splash and about to accommodate CX branding details.
- Remove un-used OWASP related image.

### CODE REVIEW

**Comment by kingthorin** [comment]
This is done. If we get alternate images I'll deal with them in another PR

**Comment by psiinon** [comment]
I think the `About` box needs to be a bit bigger - the logo and text get squeezed and theres no scrollbar.
Other than that, everything else looks good.

**Comment by kingthorin** [comment]
Better?
image

**Review by psiinon — APPROVED** [review]

File: zap/src/main/java/org/zaproxy/zap/view/AboutDialog.java [hunk 1]
```diff
@@ -66,22 +66,24 @@ public AboutDialog(Frame owner, boolean modal) {
...
-        gbcPanel.ipady = 2;
+        gbcPanel.ipady = 50;
         gbcPanel.gridwidth = 2;
 
         gbcButtons.gridx = 1;
         gbcButtons.gridy = 1;
         gbcButtons.insets = new Insets(2, 2, 2, 2);
         gbcButtons.anchor = GridBagConstraints.SOUTHEAST;
 
-        mainPanel.add(new AboutPanel(), gbcPanel);
```
Comment (thc202) [inline zap/src/main/java/org/zaproxy/zap/view/AboutDialog.java:77]: Revert?

Comment (kingthorin) [reply zap/src/main/java/org/zaproxy/zap/view/AboutDialog.java]: Done. Sorry at some point I'd been trying to set something with the panel before adding it :wink:

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**